spec/order_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'order'
+
+describe Order do
+
+	let (:order){double(:order)}
```

Comment:
Why would you create a double of the class you are describing?

Unit Test doubles exist to _isolate_ tests - so you would double any classes that _interact_ with this class.